tests/analytical_comparisons/test_analytical.py

```diff
@@ -89,4 +88,7 @@ def test_analytical(name, expected, config):
     convergence = np.log2(errs.shift() / errs).drop(index=0)
     expected_convergence = pd.Series(expected["convergence"], index=cols_err)
 
-    assert np.allclose(convergence, expected_convergence, rtol=expected.get("rtol", 1e-2))
+    # Make sure velocity and pressure have the theoretical rates
+    assert np.allclose(convergence[columns[:2]], expected_convergence[columns[:2]], rtol=expected.get("rtol", 1e-2))
+    # For normal stress we only make sure the convergence does not go bellow a minimum
+    assert np.all(convergence[columns[2]] > expected_convergence[columns[2]] - expected.get("rtol_ns", 1e-2))
```

Comment:
I expect this breaks the free surface testing... I'll fix this bit up.

Reply:
I think I have already done that; see my comment below.